lib/index.js

```diff
@@ -168,12 +171,19 @@ class PluginPassportOAuth {
     return this.getProviderRepository(profile.provider)
       .get(profile._json[this.config.strategies[profile.provider].identifierAttribute])
       .then(userObject => {
-        let req = null;
-
         if (userObject !== null) {
           return Promise.resolve({kuid: userObject.kuid, message: null});
         }
 
+        return Promise.resolve({kuid: null, message: `Could not login with strategy "${profile.provider}"`});
+      })
+      .catch(err => {
+        if (!(err instanceof NotFoundError)) {
```

Comment:
if `this.config.strategies[profile.provider].persist` is empty, we must also reject a `NotFoundError`

Reply:
If the persist object is empty then we must still allow the user to do a one time login without persisting it's user in Kuzzle. This is not a bug, this is a feature

Reply:
But as far as I understand the behaviour, it is not consistent with former implementation

Reply:
ok, now fixed in line 210 below